src/word_search_generator/__init__.py

```diff
@@ -33,12 +37,19 @@ def __init__(
             level (Optional[int], optional): Difficulty level. Defaults to None.
             size (Optional[int], optional): Puzzle size. Defaults to None.
         """
+        self._clear_solution: Puzzle = []
```

Comment:
I understand the thought behind "segregating or suppressing" the hidden words in the "Find These Words" section but I do think they should still be displayed in the answer ley. If you are looking at the answer key, you've already given up. It's also really tiny and upside down on the PDF output so you would have to really look close to see it. This would cut out the need to track a `_clear_solution` copy of the puzzle.

I can workout a new answer key output that is like below...

```
Find these words: BIRD, CAT, HORSE
* Words can go E, and S.

Answer Key: BIRD E @ (0, 4), CAT S @ (1, 7), DONKEY [HIDDEN] S @ (9, 0), HORSE E @ (4, 1)

Reply:
For longer word lists, I've found that repeated `[HIDDEN]` and `[SECRET]` look very clunky in the terminal.  Do you want me to alter the secret notification to be something subtler, like a dagger `†` or asterisk `*`, to save space?

Reply:
…or put the secret words in lowercase?

Reply:
@duck57, I just saw this particular and agree. I like the '*' option so which of the following do you prefer?

Option A (prepended): Answer Key: REGUALR E @ (2, 12), WORDS S @ (12, 1), *ZEBRA E @ (4, 11)
Option B (appended): Answer Key: REGUALR E @ (2, 12), WORDS S @ (12, 1), ZEBRA* E @ (4, 11)
Option C (appended after start): Answer Key: REGUALR E @ (2, 12), WORDS S @ (12, 1), ZEBRA E @ (4, 11)*

I prefer A myself...